janus.c

```diff
@@ -108,18 +108,9 @@ static struct janus_json_parameter admin_parameters[] = {
 	{"transaction", JSON_STRING, JANUS_JSON_PARAM_REQUIRED},
 	{"janus", JSON_STRING, JANUS_JSON_PARAM_REQUIRED}
 };
-static struct janus_json_parameter debug_parameters[] = {
```

Comment:
Setting a Boolean implies that the parameter has to be there and be of type Boolean so that validation in the helper function is appropriate.